packages/relay/src/lib/eth.ts

```diff
@@ -1211,11 +1211,23 @@ export class EthImpl implements Eth {
       ]);
       if (result) {
         if (result?.type === constants.TYPE_TOKEN) {
+          const blockInfo = await this.getBlockInfo(blockNumber, requestDetails);
+          if (blockInfo) {
+            if (parseFloat(result.entity?.created_timestamp) > parseFloat(blockInfo.timestamp.to)) {
```

Comment:
what would happen if `blockNumber` is the block number where the token was created? shouldn't we use `timestamp.from` instead?

Reply:
In that case it would return the bytecode

This condition returns `emptyHex` if we pass a blockNumber when the token/contract wasn't created, so if the creation timestamp is after `timestamp.to` then the block will be earlier. If we use `timestamp.from` it would return the empty bytecode instead

Reply:
Saw it backwards, thanks for the clarification!